espresso-support/core/src/main/java/com/novoda/espresso/ViewTestRule.java

```diff
@@ -25,11 +27,14 @@ protected Intent getActivityIntent() {
     }
 
     public void bindViewUsing(final Binder<T> binder) {
+        isIdle = false;
         getActivity().runOnUiThread(new Runnable() {
             @Override
             public void run() {
                 T view = getView();
                 binder.bind(view);
+                callback.onTransitionToIdle();
```

Comment:
Thanks for the headsup @fourlastor we had a look and what we have is very simple. The counting one looks like a layer that wouldn't add anything here.